app/components/navbar.hbs

```diff
@@ -109,9 +109,57 @@
     data-test-dropdown
     class='menu {{if this.isMenuOpen "active-menu" ""}}'
   >
-    <a data-test-profile href={{this.PROFILE_URL}} class='menu__link'>
-      My Profile
-    </a>
+    {{#if @dev}}
+      <a
+        data-test-home
```

Comment:
Suggestion:
Do you think the `data-test-` attributes should be more specific to nav? ex. `data-test-nav-home`. Because `data-test-home`, `data-test-profile` can also be used for home page or profile page. 
If you feel current one is fine, no problems.

Reply:
Done Thankyou